start_oauth_proxy.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+port=$OAUTH_PROXY_PORT
+
+uwsgi --http :${port} -w server
```

Comment:
It'd be better to run uwsgi from a config file to allow more options? (or even pass options as an argument).

If it's not needed, it can be added later.

Reply:
I don't think its needed now in running as its a simple bottle server, no configurations needed. 
Maybe if new features are added it can be updated then?